gym/vector/sync_vector_env.py

```diff
@@ -110,3 +111,6 @@ def _check_observation_spaces(self):
             "observation spaces from all environments must be "
             "equal.".format(self.single_observation_space)
         )
+
+    def render(self, mode="human", **kwargs):
+        return self.envs[0].render(mode, **kwargs)
```

Comment:
I think the `render` method of `VectorEnv` should return the rendered images for all the environment, similar to #1624, and not just the first environment. If rendering is only enabled for the first environment, I think it would make more sense to have the wrapper at the level of an individual environment. Something like:

```python
import gym

def make_env(i):
    def _make():
        env = gym.make('CartPole-v1')
        if i == 0:
            env = gym.wrappers.RecordVideo(
                env, 'videos',
                record_video_trigger=lambda x: x % 100 == 0,
                video_length=200
            )
        return env
    return _make

envs = gym.vector.AsyncVectorEnv([make_env(i) for i in range(4)])

observations = envs.reset()
for i in range(1000):
    actions = envs.action_space.sample()
    observations, rewards, dones, infos = envs.step(actions)
```

Reply:
That's a good point. I think a reasonable next step from this is to get a resolution for #1624, and then merge this PR.